pic32/cores/pic32/WInterrupts.c

```diff
@@ -255,6 +256,7 @@ void detachInterrupt(uint8_t interruptNum)
     }
 }
 
+#define VEC_ENT_CODE(I,B,P) int c = intmode[I]; IFS0bits.B=0; if (intFunc[I] != 0) (*intFunc[I])(); if(c < CHANGE && c == digitalRead(P)) { IFS0bits.B=1; }
```

Comment:
Excellent point, but I just used what I know as to be universal, as to avoid problems and complications. Fix it up :-)